server.go

```diff
@@ -265,23 +274,14 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 	conf.TLSKey = REDACTED
 	log.WithField("config", conf).Debug("Initialized server")
 
-	// Configure tracing workers and sinks
-	if len(conf.SsfAddress) > 0 && ret.tracingSinkEnabled() {
+	// Configure tracing sinks
+	if ret.tracingSinkEnabled() && len(conf.SsfListenAddresses) > 0 {
```

Comment:
Two things:
* Shouldn't this be checkint `ret.SsfListenAddrs` since only _good_ addresses will be in that, versus possible invalid ones in `conf`?
* Could we move this logic to `tracingSinkEnabled`?

Reply:
I tried making these conditions a single one, but there's some places that explicitly require that both conditions are independent of each other. (They probably shouldn't, but I didn't want to open that can of worms yet)

Reply:
Ok